externals/CMakeLists.txt

```diff
@@ -155,6 +155,19 @@ if (APPLE)
     endif()
 endif()
 
+# Dear ImGui
+block()
```

Comment:
[block()](https://cmake.org/cmake/help/latest/command/block.html) was added in CMake 3.25 but shadps4 use `cmake_minimum_required(VERSION 3.16.3)`.

Reply:
Oh, thanks for the tip. Will fix later